doc/conf.py

```diff
@@ -551,6 +553,11 @@ def setup(app):
     },
 }
 
+needs_global_options = {
```

Comment:
the results aren't getting colored in, but I'm assuming that's the latex conversion and I don't think it's worth the time to troubleshoot. The results are successfully being added to the needtable in the STR and that's the most important part.